### ISSUE TITLE
Undefined index Tokenizers/JS.php

### ISSUE THREAD

**Issue by grappler** [squizlabs/PHP_CodeSniffer#1272]
Related to #1271

When running PHPCS 2.7.1 on [html5.js](https://themes.svn.wordpress.org/kool/1.0.8/js/html5.js) I get the following PHP notice.

```
Processing html5.js PHP Notice:  Undefined index: */ in  CodeSniffer/Tokenizers/JS.php on line 730

Notice: Undefined index: */ in  CodeSniffer/Tokenizers/JS.php on line 730
[JS => 1813 tokens in 278 lines]... DONE in 38ms (0 errors, 0 warnings)
```

**Comment by gsherwood** [comment]
The file has a syntax error in it, where the first comment does not open the comment correctly, so PHPCS (and tools like JSLint) can't match the comments correctly:
```js
//**
* @preserve HTML5 Shiv 3.7.2 | @afarkas @jdalton @jon_neal @rem | MIT/GPL2 Licensed
*/
```
I've made some changes over recent versions to hide errors caused by syntax errors in PHP code, so I think it makes sense to do the same here.

**Comment by gsherwood** [comment]
I've added guard code for this specific syntax error. Thanks for reporting this.
